plugins/blockchain_statistics/plugin.cpp

```diff
@@ -18,11 +20,6 @@ namespace golos {
 namespace plugins {
 namespace blockchain_statistics {
 
-std::string increment_counter(std::string name);
-std::string increment_counter(std::string name, uint32_t value);
-std::string increment_counter(std::string name, fc::uint128_t value);
-std::string increment_counter(std::string name, share_type value);
-
 using namespace golos::protocol;
 
 struct plugin::plugin_impl final {
```

Comment:
Are you really need to track buckets for 1 minute, 1 hour and so on?
Your statistics_sender sends information only about last changes. 
You need to have previous values for operations, and don't need to have information about time periods.

Simplify your code: remove statistics from shared memory and store only runtime information with previous counters.

Reply:
@afalaleev , @kotbegemot , this heritage of old code was intended to know statistics per minute, hour, day, week or month. As we do not have API for `blockchain_statistics` plugin there are two ways: 
1. Add API and expand the functionality
2. Remove all shared memory buckets, but it causes some troubles and makes impossible to get absolutely correct statistics